contracts/token/ERC721/ERC721Token.sol

```diff
@@ -57,12 +57,16 @@ contract ERC721Token is Migratable, SupportsInterfaceWithLookup, ERC721BasicToke
   function initialize(string _name, string _symbol) public isInitializer("ERC721Token", "1.9.0") {
     name_ = _name;
     symbol_ = _symbol;
+  }
 
-    ERC721BasicToken.initialize();
-
-    // register the supported interfaces to conform to ERC721 via ERC165
-    _registerInterface(InterfaceId_ERC721Enumerable);
-    _registerInterface(InterfaceId_ERC721Metadata);
+  function _supportsInterface(bytes4 _interfaceId)
```

Comment:
A catch with ERC165, and the reason why it's implemented as a lookup table in OpenZeppelin, is that `supportsInterface` has to cost less than 30k gas. In this case we're definitely below that limit.